src/main/java/randoop/test/EmptyExceptionCheck.java

```diff
@@ -26,7 +26,9 @@ public class EmptyExceptionCheck extends ExceptionCheck {
    * <p>Appends comment indicating that catch is being ignored.
```

Comment:
The `appendCatchBehavior()` changes seem independent of the other changes in this pull request.  Can they be pull-requested separately, so that they can be merged faster and so that this pull request is simpler?